docs/griptape-framework/data/extraction-engines.md

```diff
@@ -0,0 +1,94 @@
+# Overview 
+Extraction Engines are used to extract data from text. 
```

Comment:
Let's add another sentence or two here. Maybe describe some use cases or types of extraction.

Reply:
Sure!